plugins/dev-loop/scripts/dev-loop-notify.sh

```
@@ -109,20 +108,23 @@ if [[ -n "${DEV_LOOP_TRANSCRIPT_PATH:-}" ]]; then
   message="$message | transcript: ${DEV_LOOP_TRANSCRIPT_PATH}"
 fi
 export DEV_LOOP_MESSAGE="$message"
+# Provide a pre-quoted version for safer use in templates.
+export DEV_LOOP_MESSAGE_QUOTED="'$(echo "$message" | sed "s/'/'\\\\''/g")'"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)